common/identity.go

```diff
@@ -5,14 +5,17 @@ import (
 	"net/http"
 )
 
+// ContextKey is an alias of type string
 type ContextKey string
 
+// A list of common constants used across go-ns packages
 const (
 	FlorenceHeaderKey        = "X-Florence-Token"
 	DownloadServiceHeaderKey = "X-Download-Service-Token"
 
-	AuthHeaderKey = "Authorization"
-	UserHeaderKey = "User-Identity"
+	AuthHeaderKey    = "Authorization"
+	UserHeaderKey    = "User-Identity"
+	RequestHeaderKey = "X-Request-Id"
```

Comment:
and the handler tests too :/

Reply:
I tend not to use consts like that in tests, so that the tests ensure that the consts don't end up being "" or some other brokenness leaks into the test